client/src/modules/InterviewQuestions/components/Tables/QuestionsTable/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { message, Table } from 'antd';
+import { getQuestionsColumns } from 'modules/InterviewQuestions/data/getColumns';
+import { InterviewQuestionService } from 'services/interviewQuestion';
+import { InterviewQuestion, InterviewQuestionCategory } from 'services/models';
+
+type Props = {
+  data: InterviewQuestion[];
+  handleEditQuestion: (question: InterviewQuestion) => void;
+  loadData: () => Promise<void>;
+  categories: InterviewQuestionCategory[];
+};
+
+export function QuestionsTable(props: Props) {
+  const { data, handleEditQuestion, loadData, categories } = props;
+  const interviewQuestionService = new InterviewQuestionService();
+
+  const handleDeleteQuestion = async (id: number) => {
+    try {
+      await interviewQuestionService.deleteInterviewQuestion(id);
+      await loadData();
+    } catch (error) {
+      message.error('Something went wrong. Please try again later.');
+    }
+  };
+
+  return (
+    <Table
+      scroll={{ x: 'calc(100vw - 200px)', y: 'calc(100vh - 300px)' }}
```

Comment:
do we really need it? In some cases we add it for specific functionality. Just make sure it's not copy-pase.